src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-electronic-signature-alerts.cypress.spec.js

```diff
@@ -103,12 +103,96 @@ describe('Secure Messaging Digital Signature Error flows', () => {
   });
 
   it('verify no signature alerts with auto save', () => {
-    cy.intercept(`POST`, Paths.INTERCEPT.DRAFT_AUTO_SAVE).as(`autoSaveDraft`);
-    cy.wait(`@autoSaveDraft`, { timeout: 10000 }).then(() => {
-      cy.get(Locators.ALERTS.EL_SIGN_NAME).should('not.exist');
-      cy.get(Locators.ALERTS.EL_SIGN_CHECK).should(`not.exist`);
-    });
+    cy.intercept('POST', Paths.INTERCEPT.DRAFT_AUTO_SAVE).as('autoSaveDraft');
+
+    // Trigger auto-save by making a small change
+    cy.findByTestId('message-body-field')
+      .shadow()
+      .find('#input-type-textarea')
+      .type(' additional text');
+
+    // Wait for auto-save to complete
+    cy.wait('@autoSaveDraft', { timeout: 15000 });
+
+    cy.findByText(Alerts.EL_SIGN_NAME).should('not.exist');
+    cy.findByText(Alerts.EL_SIGN_CHECK).should('not.exist');
 
     cy.injectAxeThenAxeCheck(AXE_CONTEXT);
   });
 });
+
+describe('Secure Messaging Oracle Health Digital Signature Error flows', () => {
+  beforeEach(() => {
+    SecureMessagingSite.login();
+    PatientInboxPage.loadInboxMessages();
+    PatientInboxPage.navigateToComposePage();
+    PatientComposePage.selectRecipient('VHA 649 Release of Information (ROI)');
+    PatientComposePage.selectCategory();
+    PatientComposePage.getMessageSubjectField().type(`Oracle Health DS test`, {
+      force: true,
+    });
+    PatientComposePage.getMessageBodyField().type(
+      `{home}Oracle Health signature tests`,
+      {
+        force: true,
+        moveToStart: true,
+      },
+    );
+
+    PatientComposePage.verifyElectronicSignature();
+    PatientComposePage.verifyElectronicSignatureRequired();
+  });
+
+  it(`verify user can't send Oracle Health message without electronic signature`, () => {
+    cy.get(Locators.BUTTONS.SEND).click({ force: true });
+
+    cy.get(Locators.ALERTS.EL_SIGN_NAME).should(
+      'contain.text',
+      Alerts.EL_SIGN_NAME,
+    );
+
+    cy.injectAxe();
+    cy.axeCheck(AXE_CONTEXT);
```

Comment:
you may as well use this, I see few other places where this can be applied. But not critical
```suggestion
    cy.injectAxeThenAxeCheck(AXE_CONTEXT);
```

Reply:
done!